.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Report a LWJGL bug
```

Comment:
```suggestion
about: Report an LWJGL bug
```
Small grammar nitpick, I think "LWJGL" is pronounced with a vowel sound at the start, see the README for other examples of using "an LWJGL".